admin/conf/routes

```diff
@@ -10,6 +10,7 @@ GET     /jobs/:group/:name            com.lucidchart.piezo.admin.controllers.Job
 POST    /jobs/:group/:name            com.lucidchart.piezo.admin.controllers.Jobs.putJob(group: String, name: String)
 DELETE  /jobs/:group/:name            com.lucidchart.piezo.admin.controllers.Jobs.deleteJob(group: String, name: String)
 GET     /jobs/:group/:name/editor     com.lucidchart.piezo.admin.controllers.Jobs.getEditJob(group: String, name: String)
+POST    /jobs/:group/:name/runner     com.lucidchart.piezo.admin.controllers.Jobs.runJob(group: String, name: String)
```

Comment:
What you're doing is creating a trigger, not a job. The triggerJob function just does the following:
`
        OperableTrigger trig = new org.quartz.impl.triggers.SimpleTriggerImpl(newTriggerId(),
                Scheduler.DEFAULT_GROUP, jobKey.getName(), jobKey.getGroup(),
                new Date(), null, 0, 0);
`
It seems like you should just use the existing code to create a "simple" trigger.

If you really don't want to do that and have to have a custom resource then it should at least be on /triggers because that's what's being created.